types/plotly.js/tsconfig.json

```diff
@@ -18,6 +18,8 @@
     },
     "files": [
         "index.d.ts",
-        "plotly.js-tests.ts"
+        "lib/core.d.ts",
```

Comment:
Please do not copy paste the file to create duplicate typings. 

You would want to follow something like what d3 typings do.
Please take a look at: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3/index.d.ts and the corresponding sub typings you can get eg. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-array/index.d.ts

This is what you need to do here as well.